source/help/apps/desktop-guide.rst

```diff
@@ -40,108 +40,95 @@ Editing servers and teams
+| Show red badge on taskbar icon to indicate unread messages                    | Yes                       | Yes                       |                           |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Flash taskbar icon when a new message is received                             | Yes                       |                           | Yes                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Start app on login                                                            | Yes                       |                           | Yes                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Hide menu bar                                                                 | Yes                       |                           | Yes                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Toggle window visibility when clicking on the tray icon                       | Yes                       |                           |                           |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Show icon on menu bar                                                         |                           | Yes                       | Yes                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Leave app running in notification center when application window is closed    |                           | Yes                       | Yes                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
 
 Menu Bar
 ---------------------------------------------------------------------
 
 The desktop app contains a menu bar with additional features and shortcuts to streamline your experience. 
 
-If the menu bar is hidden, you may use the ALT key to display the menu. To have the menu displayed at all times, go to the Settings page and uncheck the **Hide menu bar** setting.
+If the menu bar is hidden, you may use the ALT key to display the menu. To have the menu displayed at all times, go to the Settings page by pressing CTRL/CMD+COMMA and uncheck the **Hide menu bar** setting.
 
```

Comment:
True... Best support is no support. But it's also true that _The best help is no help_.

If the info is in the UI, then you don't need it in the help. And if people aren't sure about what one of the menu options does, then when they refer to the docs they should get more info than what's in the UI. For example, _Undo (CTRL+Z): Reverses previous action_ makes looking at the docs a waste of time, because it has no additional information. The description is basically the menu item reworded. Also, there's no one under the age of 80 who doesn't already know what Edit > Undo does.

Similarly for the other menu items, which are all pretty standard computer interface items. The only one that might be special is _Settings (CTRL+COMMA): Opens app settings where you can manage your servers and configure desktop app settings_ and it's easier to click that item and see what you get than it is to leave the app, go to the docs, and start looking around for what it means.

Crisp, tight docs without superfluous info are easier and cheaper to translate, easier to search through (because the search results are more relevant as they don't have superfluous info), and easier to read and use.